tests/data_files/Makefile

```diff
@@ -935,6 +935,9 @@ ec_prv.pk8param.pem: ec_prv.pk8param.der
 	$(OPENSSL) pkey -in $< -inform DER -out $@
 all_final += ec_prv.pk8param.pem
 
+ec_pub.der: ec_prv.sec1.der
```

Comment:
There's already a rule for `ec_pub.der` on line 1016. (And this rule is missing the `all_final +=` part.)

Also, I can't see a rule for `ec_pub.pem`. So, overall I'm think this rule should generate `ec_pub.pem` from `ec_prv.sec1.der` and then we can keep the rule on line 1016 that derives the der version from the pem version.